concepts/05 UI Components/List/00 Getting Started with List/03 Bind the List to Data.md

```diff
@@ -0,0 +1,182 @@
+The **List** can load data from different data source types. To use a local array, assign it to the [dataSource](Documentation/ApiReference/UI_Components/dxList/Configuration/#dataSource) property and specify the name of the data field you would like to display in the [displayExpr](/Documentation/ApiReference/UI_Components/dxList/Configuration/#displayExpr) property. For information on data binding with other data source types, refer to the following articles:
+
+- [JSON Data](/Documentation/Guide/UI_Components/List/Data_Binding/JSON_Data/)
+- [OData Sevice](Documentation/Guide/UI_Components/List/Data_Binding/OData_Service/)
+- [Web API service](Documentation/Guide/UI_Components/List/Data_Binding/Web_API_Service/)
+- [MongoDB Service](Documentation/Guide/UI_Components/List/Data_Binding/MongoDB_Service/)
+- [Custom Sources](/Documentation/Guide/UI_Components/List/Data_Binding/Custom_Sources//)
```

Comment:
Use links to common articles, as the DataGrid tutorial does - the inidividual articles are soon to be removed.